gtars/src/uniwig/counting.rs

```diff
@@ -176,6 +178,7 @@ pub fn core_counts(
     let mut current_end_site: (i32, i32);
 
     let mut collected_end_sites: Vec<(i32, i32)> = Vec::new();
+    let mut collected_counts: Vec<i32> = Vec::new();
```

Comment:
Does this type need to be annotated? I feel like I remember Seth arguing that we should let Rust do the type inference